apps/src/templates/teacherDashboard/SectionLoginInfo.jsx

```diff
@@ -0,0 +1,291 @@
+import React from 'react';
```

Comment:
note because this diff looks weird: i renamed `<PrintLoginCards/>` to `<SectionLoginInfo/>` and added the `<OAuthLogins/>` component to this file.

Reply:
Thanks for calling this out, and for putting the rename in a separate commit! TIL the new "hold shift + click to select a range" feature makes this kind of diff easy to see now:
<img width="388" alt="screen shot 2019-02-07 at 1 18 36 pm" src="https://user-images.githubusercontent.com/8001765/52443597-fc8f5600-2ada-11e9-940e-24e618630683.png">